ui/component/viewers/videoViewer/internal/videojs.jsx

```diff
@@ -315,26 +328,64 @@ export default React.memo<Props>(function VideoJs(props: Props) {
       // $FlowFixMe
       document.querySelector('.vjs-control-bar').style.setProperty('opacity', '1', 'important');
 
-      // change to m3u8 if applicable
-      const response = await fetch(source, { method: 'HEAD', cache: 'no-store' });
+      if (isLivestream) {
+        // $FlowFixMe
+        vjsPlayer.addClass('livestreamPlayer');
+
+        const livestreamEndpoint = `${NEW_LIVESTREAM_LIVE_API}?channel_claim_id=${userClaimId}`;
+
+        const livestreamResponse = await fetch(livestreamEndpoint, { method: 'GET' });
+
+        const livestreamData = (await livestreamResponse.json()).data;
+
+        const livestreamVideoUrl = livestreamData.VideoURL;
+
+        // const newPoster = livestreamData.ThumbnailURL;
+
+        // pretty sure it's not working
+        // vjsPlayer.poster(newPoster);
+
+        /** don't show progress bar functionality **/
+        // $FlowFixMe
+        vjsPlayer.on('play', function() {
+          // $FlowFixMe
+          vjsPlayer.liveTracker.seekToLiveEdge();
+        });
 
-      playerServerRef.current = response.headers.get('x-powered-by');
+        document.getElementsByClassName('vjs-progress-control')[0].style.display = 'none';
```

Comment:
Feels it we can hide it under `livestreamPlayer` through higher specificity, rather than forcing it here.
Something to re-visit.

Reply:
Good point

Reply:
Is the x powered by being removed or refactored somewhere else? 

We should do a few test scenarios for the regular player to make sure nothing broke with logging, view counting, Recsys, etc.